### PR TITLE
Support cross site eval in FLModelUtil

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

- Support cross site eval in FLModelUtil

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/app_common/utils/fl_model_utils.py [hunk 1]
```diff
@@ -104,27 +104,33 @@ def from_shareable(
...
+                if params_type is None:
+                    raise ValueError(f"Invalid shareable with dxo that has data kind: {dxo.data_kind}")
+                params_type = ParamsType(params_type)
+                if params_converter:
+                    dxo.data = params_converter.convert(dxo.data)
+                params = dxo.data
+        except:
+            # this only happens in cross-site eval right now
+            submit_model_name = shareable.get_header(AppConstants.SUBMIT_MODEL_NAME)
+            meta[MetaKey.SUBMIT_MODEL_NAME] = submit_model_name
```
Comment (SYangster) [inline nvflare/app_common/utils/fl_model_utils.py:128]: I see this submit_model_name is set in the meta, is this used anywhere in cse?

**Review by SYangster — APPROVED** [review]
Thanks, LGTM

File: nvflare/app_common/utils/fl_model_utils.py (hunk 1 above)
Comment (YuanTingHsieh) [reply nvflare/app_common/utils/fl_model_utils.py:128]:
```
./nvflare/app_common/executors/model_learner_executor.py:        model_name = shareable.get_header(AppConstants.SUBMIT_MODEL_NAME)
./nvflare/app_common/executors/learner_executor.py:        model_name = shareable.get_header(AppConstants.SUBMIT_MODEL_NAME)
./nvflare/app_common/ccwf/cse_client_ctl.py:            task_data.set_header(AppConstants.SUBMIT_MODEL_NAME, model_name)
```

Comment (YuanTingHsieh) [reply nvflare/app_common/utils/fl_model_utils.py:128]: so to integrate using client api, on the 3rd party scripts we need to use meta to get this model name

**Pull request merged**